app/celery/lookup_va_profile_id_task.py

```diff
@@ -47,11 +47,9 @@ def lookup_va_profile_id(
     try:
         va_profile_id = mpi_client.get_va_profile_id(notification)
 
-        # TODO #2340 - This code block causes regression failures with,
-        # "ValueError: Fernet key must be 32 url-safe base64-encoded bytes."
-        # if is_feature_enabled(FeatureFlag.PII_ENABLED):
-        #     # Encrypt the value before storage.
-        #     va_profile_id = PiiVaProfileID(va_profile_id).get_encrypted_value()
+        if is_feature_enabled(FeatureFlag.PII_ENABLED):
+            # Encrypt the value before storage.
+            va_profile_id = PiiVaProfileID(va_profile_id).get_encrypted_value()
```

Comment:
I believe the assigned values should generally be the objects themselves, then we decide what value we want at the time it's used. Unfortunately, this becomes challenging with the feature flag. You'd have to basically have two versions of this method. (with / without feature flag). This is basically written in a way that assumes the feature flag will always be there, or expects all these changes to happen later.